frontend/src/scenes/insights/views/InsightsTable/InsightsTable.tsx

```diff
@@ -88,6 +86,11 @@ export function InsightsTable({
     const { calcColumnState } = useValues(logic)
     const { setCalcColumnState } = useActions(logic)
 
+    // Only allow table aggregation options when the math is total volume otherwise double counting will happen when the math is set to uniques
+    const showTotalCount =
```

Comment:
Nit: Shouldn't this be a selector? We have a insightsTableLogic just above

Doing this in every render is _slightly_ expensive.

Reply:
How do you format the selector here? I attempted to make it dependent on props but this doesn't work because the selector won't update if props change

```
selectors: ({props}) => ({
        showTotalCount: [
            () => [],
            () => {
                return props.filters.actions?.every((entity) => entity.math === 'total' || !entity.math) && props.filters.events?.every((entity) => entity.math === 'total' || !entity.math)
            }
        ]
})
```

Reply:
There's a mechanism for creating a dependency on props. See frontend/src/lib/components/PropertyFilters/components/taxonomicPropertyFilterLogic.ts#filter, frontend/src/lib/components/VisibilitySensor/visibilitySensorLogic.tsx for examples.